features/install-cli.feature

```diff
@@ -62,6 +62,14 @@ Feature: Command behavior of client side tools installation
     Then the exit status should be 0
     And the binary should be installed in path "#{ENV['VAGRANT_HOME']}/docker"
 
+    When I evaluate and run `bundle exec vagrant service-manager install-cli docker --path #{ENV['VAGRANT_HOME']}/unknown_dir/docker`
+    Then the exit status should be 126
+    And stderr from evaluating and running "bundle exec vagrant service-manager install-cli docker --path #{ENV['VAGRANT_HOME']}/unknown_dir/docker" should match /Directory path #{ENV['VAGRANT_HOME']}/unknown_dir is invalid or doesn't exist/
```

Comment:
What's about testing/verifying that the error message in the original bug report is not in the output? Neither here, nor in any of the successful cases of install-cli